scripts/cangen/main.py

```diff
@@ -73,9 +72,9 @@ def parse():
         # pdb.set_trace()
         bus_name = bus['busName'].capitalize()
         dbc_files = bus['dbcFiles']
-
-        dbc_file_paths = [os.path.join(DIR_DBCS, dbc) for dbc in dbc_files]
-
+        
+        dbc_file_paths = [os.path.join(os.getcwd(), dbc) for dbc in dbc_files]
```

Comment:
It would also be good to pass the path through `os.path.normpath` ([docs](https://docs.python.org/3/library/os.path.html#os.path.normpath)) to collapse any relative parent directories. This will make the error messages clearer ex:

```text
Current:
FileNotFoundError: Could not find a file at ...\projects\EV5\debug\IoCheckoutFc\../../io.dbc.
With normpath()
FileNotFoundError: Could not find a file at ...\projects\EV5\io.dbc.
```